### PR TITLE
bug fix + modi

### PR DESCRIPTION
**Pull request by minerva1993**
muon eff files and globaltag fix for 2016 samples

some dataset name handling fixed (missing 'tt' from ttHJets)

weight w/o tausf implemented

Decided to use slurm for processing - one cpu per one systematic source && dataset

Exclude pdf unc for control plots - 100 more copies of histogram require high computing cost

Tau histograms from tau sel. step.

Use post-processed, plotit input histograms for yield table. Previously, pre-processed ones are used, manually scaled, with hand-written cross sections. It must be avoided.

**Under construction for better and consistent options**

### CODE REVIEW

File: nanoaodframe/postprocess.py [hunk 1]
```diff
@@ -1,127 +1,89 @@
...
+    posthist = inputf.Get('h_nevents_' + step)
+    prehist = inputf.Get('h_nevents_' + step + '__nobtag')
+    if '__btag' in inputh:
+        posthist = inputf.Get('h_nevents_' + step + '__' + str(inputh.split('__')[-1]))
     if prehist.Integral() * posthist.Integral() == 0:
-        return 1
-    return posthist.Integral() / prehist.Integral()
+        return 1.0
+    #print(prehist.Integral() / posthist.Integral())
+    return prehist.Integral() / posthist.Integral()
```
Comment (easilar) [inline nanoaodframe/postprocess.py:50]: Why did we go back to Python 2 syntax here ?

Comment (minerva1993) [reply nanoaodframe/postprocess.py:50]: I think return is just fine? Maybe I'm missing something?

Comment (easilar) [reply nanoaodframe/postprocess.py]: It is OK yes !

Comment (minerva1993) [reply nanoaodframe/postprocess.py]: @easilar lets merge this PR ONCE all dnn and limit stuffs are validated. (ie. if there is nothing missing in terms of dnn and limit setting)

**Review by easilar — APPROVED** [review]

**Pull request merged**